grakn-graph/src/main/java/ai/grakn/graph/internal/GraknComputerImpl.java

```diff
@@ -50,17 +51,18 @@
  */
 public class GraknComputerImpl implements GraknComputer {
     private final Graph graph;
-    private final Class<? extends GraphComputer> graphComputer;
+    private final Class<? extends GraphComputer> graphComputerClass;
```

Comment:
Graph computer contains hadoopConfiguration and reference to the preloaded spark rdd. So it is better to create it lazily.